backend/src/Models/Elections.ts

```diff
@@ -72,15 +85,16 @@ export default class ElectionsDB {
 
         let querry = this._postgresClient
             .selectFrom(tableName)
+            .where('head', '=', true)
             .selectAll()
+
         if (id !== '' || email !== '') {
-            querry = querry.where(({ or, cmpr }) =>
-                or([
-                    cmpr('owner_id', '=', id),
-                    cmpr(sql`admin_ids::jsonb`, '?', email),
-                    cmpr(sql`audit_ids::jsonb`, '?', email),
-                    cmpr(sql`credential_ids::jsonb`, '?', email)
-                ]))
+            querry = querry.where(({ eb }) =>
```

Comment:
typo? s/querry/query

Reply:
Ah, I kept making that mistake. I'll do a search to make sure I didn't do that anywhere else.

Reply:
Fixed